inference-engine/src/inference_engine/generic_ie.cpp

```diff
@@ -28,6 +28,8 @@ void ngraph::op::GenericIE::addExtension(std::shared_ptr<const ngraph::Function>
 
     for (auto r : func->get_results())
         nodes.emplace_back(r);
+    for (auto s : func->get_sinks())
+        nodes.emplace_back(s);
     for (auto param : func->get_parameters())
         nodes.emplace_back(param);
```

Comment:
If we need sinks to `traverse_nodes`, we need also update method `traverse_nodes` which works with nGraph Function.

Reply:
@ilyachur could you point out where is it? I have fixed everything that I have found

Reply:
Please take a look to methods from this file: https://github.com/openvinotoolkit/openvino/blob/4d2674962d2518f094d37cc5f2f401aa95cbfb69/ngraph/core/src/graph_util.cpp#L54

Reply:
traverse_nodes in this file already use sinks